templates/consulting/index.html

```diff
@@ -0,0 +1,515 @@
+            </p>
+          </div>
+          <div class="p-equal-height-row__col">
+            <p class="p-heading--5 p-equal-height-row__item">
+              We know
+              <br class="u-hide--small" />
+              the technologies inside-out
+            </p>
+            <p class="p-equal-height-row__item">
+              If you choose Canonical's open source technologies, you can rest assured in our delivery as we know them best. We distribute, package and secure our own products. We are the true authority in delivering them to our customers.
+            </p>
+          </div>
+          <div class="p-equal-height-row__col">
+            <p class="p-heading--5 p-equal-height-row__item">
+              We bring
+              <br class="u-hide--small" />
+              proven delivery expertise
+            </p>
+            <p class="p-equal-height-row__item">
+              We have been delivering open source infrastructure and applications for over 20 years, supporting thousands of customers across the full compute spectrum.
+            </p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="p-section--shallow">
+      <div class="u-fixed-width">
+        <hr class="p-rule--highlight" />
+      </div>
+      <div class="row">
+        <div class="col-3 col-medium-2">
+          <h2 class="p-muted-heading">What our customers say</h2>
+        </div>
+      </div>
+    </div>
+
+    <div class="row p-section--shallow">
+      <div class="col-3 col-medium-2 u-hide--small">
+        <div class="p-section--shallow">
+          {{ image(url="https://assets.ubuntu.com/v1/98a3af7e-esa.png",
+                    alt="",
+                    width="568",
+                    height="101",
+                    hi_def=True,
+                    loading="lazy") | safe
+          }}
+        </div>
+      </div>
+      <div class="col-9 col-start-large-4 col-small-6">
+        <hr class="p-rule--muted" />
+        <div class="row">
+          <div class="col-6">
+            <div class="p-section--shallow">
+              <p class="p-heading--4">
+                <i>
+                  “For ESA it is key to have service providers who can support its ambitious space programme into the 2030s and beyond to help secure Europe’s role in space exploration.”
+                </i>
+              </p>
+            </div>
+          </div>
+          <div class="col-3">
+            <hr class="p-rule--muted u-hide--large" />
+            <p>
+              <strong>Michael Hawkshaw</strong>
+              <br />
+              <span class="u-text--muted">
+                Mission Operations Infrastructure IT Service Manager
+                <br />
+                European Space Agency
+              </span>
+            </p>
+          </div>
+        </div>
+        <div class="p-cta-block">
+          <a href="/case-study/esa">Explore the full case study ›</a>
```

Comment:
We should be using html entities for the symbols. Can you update them across the file please?
```suggestion
          <a href="/case-study/esa">Explore the full case study&nbsp;&rsaquo;</a>
```